app/src/main/res/layout/activity_main.xml

```diff
@@ -1,47 +1,50 @@
 <RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
-    xmlns:tools="http://schemas.android.com/tools" android:layout_width="match_parent"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
     android:layout_height="match_parent"
     tools:context=".MainActivity"
     android:background="#333333">
 
-    <ListView
-        android:layout_width="fill_parent"
-        android:layout_height="wrap_content"
-        android:id="@+id/listView"
-        android:layout_above="@+id/linearLayout" />
-
```

Comment:
Moved this below the LinearLayout

when you reference things like layout_above, the view with the ID you're referring to has to be written first in the layout, or the reference won't work